OpenRA.Mods.Common/Traits/Render/WithInfantryBody.cs

```diff
@@ -46,7 +46,7 @@ public int QuantizedBodyFacings(ActorInfo ai, SequenceProvider sequenceProvider,
 		}
 	}
 
-	public class WithInfantryBody : ITick, INotifyAttack, INotifyIdle
+	public class WithInfantryBody : UpgradableTrait<WithInfantryBodyInfo>, ITick, INotifyAttack, INotifyIdle
 	{
 		readonly WithInfantryBodyInfo info;
```

Comment:
This can be removed (`UpgradableTrait<WithInfantryBodyInfo>` provides a public Info field).